src/components/navigation/mobile-menu.tsx

```diff
@@ -21,42 +21,64 @@ interface MobileMenuLink {
 
 export function MobileMenu(props: MobileMenuProps) {
   const { links } = props
+
+  const [openEntry, setOpenEntry] = React.useState(-1)
+
+  function toggle(
+    e: React.MouseEvent<HTMLAnchorElement, MouseEvent>,
+    index: number
+  ) {
+    e.preventDefault()
+    if (index === openEntry) setOpenEntry(-1)
```

Comment:
I'd really prefer `number | undefined` or `number | null` over some magical `-1` value. The typing of `number | undefined` (or `null`) makes it obvious that there is a special case I need to handle. On the other hand, I can pass `-1` just fine to any other function / component that accepts `number` (which would most likely be a bug in this case).

Reply:
check, will keep that in mind.